README.md

```diff
@@ -20,7 +20,72 @@
 
 ### configuration and population
 
+The config file config.py got a lot of interesting setting:
 
+```python
+population_file_base="e2pieces.txt"
+NGEN = 2000
+mutate_inpd=0.01
+selection_ind_value_step=1
+elitism_percentage_start=10
+elitism_percentage_up=4
+gen_modulo_elitism=100
+```
+
+- population_file_base:
+- NGEN:
+
+### Workflows. Running Scripts
+
+  The workflow starts by running the script called main then the script render
+
+  ```bash

```

Comment:
I agree with you for each command line used, which is well shell code. But the lines following the commands are only text output, and does not have any native color rendering, nor on Linux nor on Windows.

The best thing I think is to use both rendering styles.

### Also why not using titles to pack together each command + output
```sh
  $ python main.py
```
```
  [Error 183] Cannot create a file when that file already exists: './gen/'
  Personal Path used for this Puzzle: gen/lumy_13-02-2017_13h.14m.46s/
  No Solution Look at the logbook.
```
### Title foobar
```sh
  $ python render.py gen/lumy_13-02-2017_13h.14m.46s/
```
```
  (it take few time before an ouput arrive because it's loading big binary file).
  progression Bar 1 #################
  progression Bar 2 ########________#
```
### Title foooobaz
```sh
  $ ./gen/generations_images_to_video.bash gen/lumy_13-02-2017_13h.14m.46s/ # Only linux
```